packages/lists/init.lua

```diff
@@ -280,16 +280,18 @@ end
 package.documentation = [[
 \begin{document}
 \font:add-fallback[family=Symbola]% HACK Gentium Plus (SILE default font) lacks the circle bullet :(
-The \autodoc:package{lists} package provides enumerations and bullet lists (a.k.a. \em{itemization}\kern[width=0.1em]) which can be nested together.
+The \autodoc:package{lists} package provides enumerated and itemized (also known as \em{bulleted lists}) which can be nested together.
 
 \smallskip
-\em{Bullet lists.}
+\noindent
+\em{Itemized lists}
```

Comment:
Out of curiosity, is "itemized lists" more understandable that "bullet lists" for casual readers (not necessarily used to LaTeX etc.)? Being French, I can't say, but I had the feeling that "bullet lists" was more clear maybe. In French we'd likely use "listes à puces" - Google Translate translates it as "bulleted lists"; but again, I don't know what's best for English readers.

Reply:
I'd say that for people who are not typesetters (or LaTeX nerds), the English term would be 'bulleted lists' (and not 'bullet lists'). But because the package itself uses 'itemize,' I figured that was a better primary term to use. Hence why I put 'bulleted lists' as the 'aka' clause as opposed to the other way around.